### PR TITLE
Add attribute 'tier' to Environment model

### PR DESCRIPTION
**Pull request by Rusev11**
- Add attribute `tier` to Environment model
- Update `createEnvironment` and `updateEnvironment` in EnvironmentApi

### CODE REVIEW

**Comment by Rusev11** [comment]
@jmini Could you please review this Pull request?

File: src/main/java/org/gitlab4j/api/models/Environment.java
Comment (jmini) [inline src/main/java/org/gitlab4j/api/models/Environment.java:1]: Please add the attribute in the `src/test/resources/org/gitlab4j/api/environment.json` used in the `TestGitLabApiBeans` test class.

Comment (Rusev11) [reply src/main/java/org/gitlab4j/api/models/Environment.java:1]: Done

**Comment by Rusev11** [comment]
Should I bump the version?

**Comment by jmini** [comment]
I added a backward compatible method (with a `@Deprecated` annotation) on `main` and I removed it on `6.x`

**Pull request merged**